libs/brain/command/src/lib/brn-command.directive.ts

```diff
@@ -76,6 +82,13 @@ export class BrnCommandDirective implements AfterViewInit {
 			},
 			{ allowSignalWrites: true },
 		);
+
+		this.keyManager.change.pipe(takeUntilDestroyed()).subscribe(() => {
+			const value = (this.keyManager.activeItem as BrnCommandItemDirective)?.value();
```

Comment:
I think this casting can be removed as the type or `BrnCommandItemDirective | null` is inferred and we are handling the null case with `?`:

```suggestion
    const value = this.keyManager.activeItem?.value();
```

Reply:
you are right, thanks for pointing that out